com.unity.ml-agents/Runtime/SideChannel/RawBytesChannel.cs

```diff
@@ -13,12 +12,7 @@ public class RawBytesChannel : SideChannel
         /// the same on Python and Unity.</param>
         public RawBytesChannel(int channelId = 0)
         {
-            m_ChannelId = channelId;
-        }
-
-        public override int ChannelType()
-        {
-            return (int)SideChannelType.RawBytesChannelStart + m_ChannelId;
+            ChannelId = (int)ReservedChannelId.RawBytesChannelStart + channelId;
```

Comment:
Not sure what we want to do here - either rename the ctor argument to e.g. `rawBytesChannelOffset`, or just use the arg directly without offsetting by `RawBytesChannelStart`. I think I prefer the latter; it's a behavior change from current version, but is more consistent with what I'm proposing on FloatChannel ctor.

We could have an extra check that `channelId >= ReservedChannelId.RawBytesChannelStart && channelId < ReservedChannelId.RawBytesChannelMax` if we're worried about collisions.

Reply:
I made this change